controller/controller.go

```diff
@@ -53,6 +58,13 @@ const (
 	loggerKeyVersion    = "version"
 )
 
+var (
+	defaultPauseAnnotations = map[string]string{
+		"cluster.x-k8s.io/paused":          "true",
+		"operatorkit.giantswarm.io/paused": "true",
```

Comment:
What would be the use case that requires a different annotation than the one used upstream?

Reply:
`operatorkit` is an abstract framework that can be used for all kinds of operators. CAPI is a niche within the Kubernetes ecosystem. When you build an nginx operator why would you want to have to know anything about Cluster API? I mean, I would not want to. I want to build my nginx operator and maybe use the pausing annotation for my field of expertise. 🤗